src/account/NexusSmartAccount.ts

```diff
@@ -1881,13 +1889,13 @@ export class NexusSmartAccount extends BaseSmartContractAccount {
       | Awaited<ReturnType<typeof this.getInstalledExecutors>>,
     module: Module
   ): Hex {
-    const index = installedModules.indexOf(getAddress(module.moduleAddress))
+    const index = installedModules[0].indexOf(getAddress(module.moduleAddress))
     if (index === 0) {
       return SENTINEL_ADDRESS
     }
     if (index > 0) {
       // @ts-ignore: TODO: Gabi This looks wrong
```

Comment:
check this comment. if addressed please delete it

Reply:
Let's chat about this on a call

Reply:
This is actually how rhinestone is handling it in their sdk also @livingrockrises